win32ss/user/user32/windows/cursoricon.c

```diff
@@ -170,8 +173,19 @@ static int bitmap_info_size( const BITMAPINFO * info, WORD coloruse )
                 colors = 256;
         if (!colors && (info->bmiHeader.biBitCount <= 8))
             colors = 1 << info->bmiHeader.biBitCount;
+        /* Account for BI_BITFIELDS in BITMAPINFOHEADER(v1-v3) bmp's. The
+         * 'max' selection using biSize below will exclude v4 & v5's. */
         if (info->bmiHeader.biCompression == BI_BITFIELDS) masks = 3;
         size = max( info->bmiHeader.biSize, sizeof(BITMAPINFOHEADER) + masks * sizeof(DWORD) );
+        /* Test if compression BI_BITFIELDS and bpp either 16 or 32.
```

Comment:
Fixing english:
```suggestion
        /* Test for BI_BITFIELDS format and either 16 or 32 bpp.
```
(BI_BITFIELDS means _without_ compression by the way)

Reply:
Changed. Thanks.